lightningd/options.c

```diff
@@ -244,19 +244,13 @@ if (!parse_tor_wireaddr(arg,(u8 *)(ld->tor_proxy_ip),(u16 *)&ld->tor_proxy_port)
 return NULL;
 }
 
+
+
 static char *opt_add_tor_hidden_service(const char *arg, struct lightningd *ld)
 {
 
-//FIXME: TODO SAIBATO
-/*
- * check service port active
- * do we need here async ? no just open socket sync
- * AUTHENTICATE
- * CREATE TEMP ONION ADDR
- * display to user
- * done
- *
- */
+if (strstr(arg,"true"))
```

Comment:
Best to set a flag in `struct lightningd`, then process flag in `main` function (`if (ld->make_onion_service) create_tor_hidden_service_conn(ld);`).  For example, if we set this flag in configuration file but we want to override this temporarily on a run for debugging.